plinth/modules/bind/__init__.py

```diff
@@ -31,7 +31,9 @@
 
 version = 1
 
-title = _('Domain Name Server \n (BIND)')
+title = _('BIND')
```

Comment:
Since the title is no longer the title of the page, shouldn't we call it `name` instead?